### PR TITLE
Added DGB support

### LINKED ISSUE #104: Add support for DGB addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

### PR DESCRIPTION
**Pull request by surajsingla333**
## Issue number

closes #104 

## Description

The PR is related to the gitcoin bounty, https://gitcoin.co/issue/ensdomains/address-encoder/104/100023811,
As per the description, DGB support is added

## List of features added/changed

- Added DGB support using base58check P2PKH and P2SH

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.

## How Has This Been Tested?

- Using base58check to generate hex format of DGB addresses

## Screenshots (if appropriate):

Screenshot (3157)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

**Comment by surajsingla333** [comment]
Hey @Arachnid, the changes have been made and the branch is ready to be merged.

File: src/index.ts [hunk 1]
```diff
@@ -753,12 +753,12 @@ function aionEncoder(data: Buffer): string {
 }
 
 const getConfig = (name: string, coinType: number, encoder: EnCoder, decoder: DeCoder) => {
-  return {
-    coinType,
-    decoder,
-    encoder,
+    return {
```
Comment (makoto) [inline src/index.ts:756]: Please remove extra white spaces

**Comment by makoto** [comment]
`NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.` part is missing. Please specify where you found the encoding type, version number and test data so that I can verify

**Comment by makoto** [comment]
Hi, please let me know if you want to continue working on this or you want me to pass on to someone else.

**Pull request closed without merging**